docs/sources/alerting/unified-alerting/grafana-managed-number-alerts.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Numeric data Grafana manged alert rules"
+description = "Number data based Grafana manged alert rules"
+keywords = ["grafana", "alerting", "guide", "rules", "create"]
+weight = 400
++++
+
+# Alerting on numeric data
+
+With certain data sources number data that is not time series can be directly alerted on, or passed into Server Side Expressions (SSE). This allows for more efficiency by doing more processing within the data source, and can also simplify alert rules.
+
+When alerting on numeric data instead of time series data, there is no need to reduce each labeled time series into a single number. Instead labeled numbers are returned to Grafana instead.
+
+## Tabular Data
+
+This feature is supported with backend data sources that query tabular data:
+
+- SQL data sources such as MySQL, Postgres, MSSQL, and Oracle.
+- The Azure Kusto based services: Azure Monitor (Logs), Azure Monitor (Azure Resource Graph), and Azure Data Explorer.
+
+A query with Grafana managed alerts or SSE is considered numeric with these data sources, if:
+
+- The "Format AS" option is set to "Table" in the data source query.
+- The table response returned to Grafana from the query includes only one numeric (e.g. int, double, float) column, and optionally additional string columns.
+
+If there are string columns then those columns become labels. The name of column becomes the label name, and the value for each row becomes the value of the corresponding label. If multiple rows are returned, then each row should be uniquely identified their labels.
+
+## Example
+
+If you have a MySQL table called "DiskSpace" like the following:
```

Comment:
It would be nice to include also a description of this alert like: fire when at least one server disk has lower disk space than 5%.